File: app/src/main/java/com/lumiyaviewer/lumiya/voice/common/VoicePluginMessenger.java
package com.lumiyaviewer.lumiya.voice.common;

import android.os.Bundle;
import android.os.Handler;
import android.os.Message;
import android.os.Messenger;
import android.os.RemoteException;

public class VoicePluginMessenger {
    public static boolean sendMessage(Messenger messenger, VoicePluginMessageType voicePluginMessageType, VoicePluginMessage voicePluginMessage, Messenger messenger2) {
        if (messenger == null) {
            return false;
        }
        Bundle bundle = new Bundle();
        bundle.putString("messageType", voicePluginMessageType.toString());
        bundle.putBundle("message", voicePluginMessage.toBundle());
        Message obtain = Message.obtain((Handler) null, 200, bundle);
        obtain.replyTo = messenger2;
        try {
            messenger.send(obtain);
            return true;
        } catch (RemoteException e) {
            return false;
        }
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/voice/common/VoicePluginMessenger.kt
package com.linkpoint.voice.common

import android.os.Bundle
import android.os.Message
import android.os.Messenger
import android.os.RemoteException
import com.linkpoint.voice.common.VoicePluginMessage
import com.linkpoint.voice.common.VoicePluginMessageType

class VoicePluginMessenger {
    /*
     * Enabled force condition propagation
     * Lifted jumps to return sites
     */
    @JvmStatic
     fun sendMessage(messenger: Messenger, voicePluginMessageType: VoicePluginMessageType, voicePluginMessage: VoicePluginMessage, messenger2: Messenger): Boolean {
        Boolean bl
        val bl2: Boolean = bl = false
        if (messenger == null) return bl2
        val bundle: Bundle = Bundle()
        bundle.putString("messageType", voicePluginMessageType.toString())
        bundle.putBundle("message", voicePluginMessage.toBundle())
        voicePluginMessageType = Message.obtain(null, (Int)200, (Object)bundle)
        ((Message)voicePluginMessageType).replyTo = messenger2
        try {
            messenger.send((Message)voicePluginMessageType)
            return true
        }
        catch (RemoteException remoteException) {
            return bl
        }
    }
}

